spec/contiv/libraries/netmaster.raml

```diff
@@ -1,186 +1,217 @@
 #%RAML 1.0 Library
```

Comment:
Which version of Ruby did you use to generate the initial RAML file?

It looks like YAML now requires certain strings to be explicitly quoted (which necessitates escaping things) and the keys also got reordered which is what is causing the huge diff.

Since we're now using a container to regenerate this with a fixed version of Ruby, we shouldn't have any big diffs in the future so I think it's okay to leave it in.